### ISSUE TITLE
Protobuf Support on Solaris

### ISSUE THREAD

**Issue by AyberkSorgun** [hazelcast/hazelcast#19047]
More info - https://github.com/hazelcast/hazelcast/issues/18767#issuecomment-860661606

Protobuf is used by `hazelcast-jet-kafka` and there is no protobuf lib for solaris, hence we can't support it, unless we find a workaround.

we should fix the solaris jenkins build so that it skips the `hazelcast-jet-kafka` extension, cause right now we get zero tests run and the jenkins job is disabled

**Comment by frant-hartm** [comment]
@Serdaro could you please update the relevant section of the documentation.

The modules not supported on solaris are:

hazelcast-jet-grpc
hazelcast-jet-protobuf
hazelcast-jet-python

hazelcast-jet-kafka is supported, so you can remove this one. Sorry for the confusion. CC @AyberkSorgun

**Comment by frant-hartm** [comment]
Closed by the PR, but the docs still need to be updated.

**Comment by mmedenjak** [comment]
Looks like the [docs contain the note](https://docs.hazelcast.com/hazelcast/5.0-beta-1/sql/kafka-connector.html#installation). Moving this issue to backlog so we may assess investing into solaris support in the future.

**Comment by mmedenjak** [comment]
Also, @olukas @slavagithub I see the solaris job is re-enabled so you can remove the "disabled" note here:
http://jenkins.hazelcast.com/job/Hazelcast-master-Solaris-OracleJDK8/

And remove this job - http://jenkins.hazelcast.com/job/Hazelcast-master-Solaris-OracleJDK8-without-checkstyle/

**Comment by Holmistr** [comment]
@mmedenjak I'm confused now, anything needed from me? :)

**Comment by slavagithub** [comment]
> Also, @olukas @slavagithub I see the solaris job is re-enabled so you can remove the "disabled" note here:
> http://jenkins.hazelcast.com/job/Hazelcast-master-Solaris-OracleJDK8/
> 
> And remove this job - http://jenkins.hazelcast.com/job/Hazelcast-master-Solaris-OracleJDK8-without-checkstyle/

@mmedenjak @olukas http://jenkins.hazelcast.com/job/Hazelcast-master-Solaris-OracleJDK8-without-checkstyle/ is disabled. http://jenkins.hazelcast.com/job/Hazelcast-master-Solaris-OracleJDK8/ is enabled, note is removed

**Comment by mmedenjak** [comment]
@Holmistr no, sorry, not really. I thought this issue was to add solaris support, so I decided to move it to backlog, and then I realised that the documentation needs to be amended so I moved it back (after which we can move it to backlog since there's no point in working on this any further in 5.0).

**Comment by mmedenjak** [comment]
We agreed this should be moved to documentation scope as right now the only thing left is to document the limitations. Adding support for solaris is currently in the far future, if ever.
